### DISCUSSION TITLE
Error occurs when using API(com.google.cloud.speech.v1p1beta1.SpeechClient)

### DISCUSSION THREAD

**Discussion by hobj** [community/community#79076] [Other Feature Feedback, Questions, & Ideas]
Exception in thread "main" java.util.ServiceConfigurationError: io.grpc.LoadBalancerProvider: io.grpc.xds.WeightedRoundRobinLoadBalancerProvider Unable to get public no-arg constructor

Is there a solution to why this error occurs?

**Comment by rhuthik** [comment]
This error typically occurs due to a misconfiguration in the gRPC service loader setup. First, ensure that the META-INF/services directory in your JAR file correctly lists the io.grpc.LoadBalancerProvider with a properly formatted file.Also check that the WeightedRoundRobinLoadBalancerProvider class has a public, no-argument constructor, as it's required for service provider instantiation via reflection and then verify that all gRPC dependencies are correctly included and compatible with each other in your project's build configuration.
